src/Core/Binding/IBindingConfigProvider.cs

```diff
@@ -44,7 +45,7 @@ public interface IBindingConfigProvider
     /// For C++ it will be in a json file in a Sonar-specific format</remarks>
     public interface IBindingConfig
     {
-        string FilePath { get; }
+        IList<string> SolutionItems { get; }
```

Comment:
`SolutionItems` is not a very descriptive name - it already has a specific meaning in VS, and it doesn't convey any information about the content of the strings. Something like `SolutionLevelFilePaths` would be better.

A doc comment would be useful: it's a list of all of the solution-level files that should exist if the solution is correctly bound. It doesn't say whether the files exist or not. It would also be useful to state whether the value can be null.